libs/utils/env.py

```diff
@@ -203,8 +207,13 @@ def __init__(self, target_conf=None, test_conf=None, wipe=True,
             self._log.info('Loading custom (file) target configuration')
             self.conf = self.loadTargetConfig(target_conf)
         elif target_conf is None:
-            self._log.info('Loading default (file) target configuration')
-            self.conf = self.loadTargetConfig()
+            env_target_conf = os.environ.get('LISA_TARGET_CONF')
```

Comment:
Not sure to get your point here... what I'm proposing above is just to use the ENV variable to override the target.conf file to use.

Reply:
I'll change self.loadTargetConfig so that these 2 are equivalents:
```self.loadTargetConfig()
self.loadTargetConfig("")
```
This way, this code will work as you expect it:
```
env_target_conf = os.environ.get('LISA_TARGET_CONF', '')
self.conf = self.loadTargetConfig(env_target_conf)
```